### ISSUE TITLE
What's this crate's minimum support rust policy?

### ISSUE THREAD

**Issue by psychon** [esposm03/xcursor-rs#20]
Hi there,

[I got a request to drop my own xcursor parsing code and use yours for code-deduplication](https://github.com/psychon/x11rb/issues/934). The biggest risk that I see with this plan is MSRV bumps. In the past, x11rb bumped its MSRV because its dependencies did, e.g. https://github.com/psychon/x11rb/commit/a20217f208fc40789922a5a12b55851de80cd057#diff-31339aff40c3ee5a01610a9b9aebfe3b9e30f8f54d1ccf6f2dd566097b5ecc39 and https://github.com/psychon/x11rb/commit/3c209ad8fd69b9ee8aa186a0c04c02849c577466

For that reason, I am curious about what your thoughts about MSRV are? Is that futile and people should always use the latest and greatest? Would you accept a PR that adds a `rust-version` field to `Cargo.toml? Would you accept a PR that actively tests the MSRV by trying to build with that Rust version in Github Workflows? Do you need care about MSRV?

Basically, I'm interested on your position. If necessary, I can then e.g. do PRs that add a workflow for MSRV testing, if you are interested.

**Comment by esposm03** [comment]
Hi, I didn't actually know what the MSRV was, so I bisected a bit the compiler versions and found out it is Rust 1.34, which is _ancient_ (it was released in april 2019). However, since this is a small library with zero dependencies, and which I don't expect to need many changes in the future, I don't expect to need to change that.

> For that reason, I am curious about what your thoughts about MSRV are?

I think it's useful generally, however I don't think that it makes much of a difference here. The crate is still on edition 2018, and the `rust-version` field you mentioned is only read with a Cargo version that is at least 1.56, which is actually the same version that stabilised edition 2021; so if your Cargo reads that field, you are guaranteed to be able to build this crate.

> If necessary, I can then e.g. do PRs that add a workflow for MSRV testing, if you are interested.

I don't think it's necessary, however if you feel like preparing such a PR, go for it and I will accept it
